### DISCUSSION TITLE
Support for C1152 Global Grid?

### DISCUSSION THREAD

**Discussion by mdtoyNOAA** [NOAA-EMC/global-workflow#1824] [General]
I have been running the global workflow on Hera at C384 and C768 resolutions.  I tried to set up a C1152 experiment, but this resolution is not supported on Hera?  Is there any way at all to run the workflow at C1152?  Thank you.

**Comment by JessicaMeixner-NOAA** [comment]
@mdtoyNOAA I've looked before and saw that C1152 fix files were there.  If you have an IC I would think you could run, but haven't tried it yet. Hoping to do so in a couple of weeks.

**Reply by mdtoyNOAA** [reply]
I too saw that the C1152 files were available.  The error message I got when running the script ./workflow/setup_expt.py is:
NotImplementedError: Supported resolutions on HERA are:
C768, C384, C192, C96, C48

I could try adding 'C1152' to the "SUPPORTED_RESOLUTIONS" list in hosts/hera.haml and see if it works.

**Reply by mdtoyNOAA** [reply]
@JessicaMeixner-NOAA   Adding 'C1152' to the "SUPPORTED_RESOLUTIONS" list in hosts/hera.yaml worked.  Both 'setup_expt.py' and 'setup_xml.py' scripts ran without complaint.  Thank you.

**Reply by JessicaMeixner-NOAA** [reply]
That's good to hear!

**Comment by aerorahul** [comment]
user reports adding necessary keys in the right place resolved this issue.
